UnityProject/Assets/Scripts/Core/Input System/InteractionV2/Validations.cs

```diff
@@ -772,4 +772,83 @@ public static bool HasBothHands(GameObject player)
 
 		return health.HasBodyPart(BodyPartType.LeftArm, true) && health.HasBodyPart(BodyPartType.RightArm, true);
 	}
+
+
+	/// <summary>
+	/// Validates that the player's character name.
+	/// </summary>
+	/// <param name="characterName">CharacterSettings.Name</param>
+	/// <returns>True if illegal.</returns>
+	public static bool HasIllegalCharacterName(String characterName)
```

Comment:
kind of funny place to put it since it's under interactions  "ore/Input System/InteractionV2/Validations.cs "

Reply:
Moved to server validations.